src/lib/helpers/getInput.ts

```diff
@@ -0,0 +1,30 @@
+import type { Locator } from 'playwright-chromium';
+
+import type { BrowserPage } from 'lib/browser/Page';
+import type { HandledError } from 'lib/types';
+
+/**
+ * Get input for selector.
+ */
+export async function getInput(
+  page: BrowserPage | undefined,
+  sel: string
+): Promise<Locator | { error: HandledError; rawError: Error }> {
+  const textInputLoc = page?.ref?.locator(sel);
+  const count = textInputLoc ? await textInputLoc.count() : 0;
+  if (!textInputLoc || count <= 0) {
+    return {
+      error: 'field_not_found',
+      rawError: new Error(sel),
```

Comment:
Can you put a more explicit message? I suppose both `error` and `rawError` are meant to stay together, but still it's weird to have just a CSS selector as error message.